src/containers/DebitCard/bridgeCard/cardRevealAuth.tsx

```diff
@@ -124,7 +124,7 @@ export default function CardRevealAuthScreen(props: {
           showModal('state', {
             type: 'error',
             title: t('VERIFICATION_FAILED'),
-            description: t('INVALID_OTP'),
+            description: response?.error?.message ?? t('INVALID_OTP'),
```

Comment:
use parseErrorMessage function from utils instead of response?.error?.message do parseErrorMessage(response.error)